apps/SageAccountWeb/src/components/RegisterAccount1.tsx

```diff
@@ -148,36 +170,31 @@ export const RegisterAccount1 = () => {
         alias: username,
         type: AliasType.USER_NAME,
       })
-      if (!aliasCheckResponse.available) {
-        displayToast('Sorry, that username has already been taken.', 'danger')
-      } else if (!aliasCheckResponse.valid) {
-        displayToast('Sorry, that username is not valid.', 'danger')
-      } else {
-        // Looks good!  Go to Google oauth account creation flow
-        // redirect to Google login, passing the username through via the state param.
-        // Send us back to the special oauth2 account creation step2 path (which is ignored by our AppInitializer)
-        localStorage.setItem(
-          SynapseConstants.LAST_PLACE_LOCALSTORAGE_KEY,
-          `${SynapseClient.getRootURL()}authenticated/signTermsOfUse`,
-        )
-        const redirectUrl = `${SynapseClient.getRootURL()}?provider=${
-          SynapseConstants.OAUTH2_PROVIDERS.GOOGLE
-        }`
-        SynapseClient.oAuthUrlRequest(
-          SynapseConstants.OAUTH2_PROVIDERS.GOOGLE,
-          redirectUrl,
-          { registrationUsername: username },
-        )
-          .then((data: any) => {
-            const authUrl = data.authorizationUrl
-            window.location.assign(authUrl)
-          })
-          .catch((err: any) => {
-            displayToast(err.reason as string, 'danger')
-          })
```

Comment:
We're already using `try`/`catch` here, so change this to an `await`

Reply:
I'm confused by this comment.  Are you referring to the await SynapseClient.isAliasAvailable()?

Reply:
I just mean `SynapseClient.oAuthUrlRequest` was written using `.then(...).catch(...)` when the error handling behavior inside the `catch` was identical to the error handling behavior in the `try/catch` that is already wrapping this code block.

So we get the same behavior with less, cleaner code by using `await` instead of promise callbacks